README.md

```diff
@@ -101,6 +101,27 @@ var advancedOptions = new AdvancedOptions()
 var httpClient = HttpClients.NewHttpClient(cassette, Mode.Replay, advancedSettings);
 ```
 
+### Expiration
+
+Set expiration dates for recorded requests, and decide what to do with expired recordings.
+
+**Default**: *No expiration*
+
+```csharp
+using EasyVCR;
+
+var cassette = new Cassette("path/to/cassettes", "my_cassette");
+var advancedOptions = new AdvancedOptions()
+{
+    ValidTimeFrame = new TimeFrame() {  // Any matching request is considered expired if it was recorded more than 30 days ago
+        Days = 30,
+    },
+    WhenExpired = ExpirationActions.Address // Throw exception if in replay mode, automatically re-record if in auto mode
```

Comment:
Thought: I think having this cleverly switch behaviors depending on the mode you're in is an antipattern. I'd expect the option I select to do one thing only. I wonder if we can't split these up to be more explicit with namings to avoid confusion and this side-effect.

Reply:
We probably could, but I'd have to do some validation to make sure you haven't set an invalid settings combo (i.e. we can't automatically re-record when in `Replay` mode)

Reply:
Resolved

Reply:
Issue: The examples here, namely `.Address` here was fixed in code but not in the example.

Reply:
Resolved